docs/miner.md

```diff
@@ -36,6 +36,13 @@ cd pretraining
 python -m pip install -e .
 ```
 
+Note: flash-attn may not have their dependencies set up correctly. If you run into issues try installing those requirements separately first:
+```shell
+pip install packaging
```

Comment:
Should we just add these to requirements?

Reply:
Pip doesn't support ordering of requirements in a semantically meaningful way. It expects packages to have requirements correctly identified and flash-attn has said this is too much hassle for them.

https://github.com/Dao-AILab/flash-attention/issues/246
https://github.com/Dao-AILab/flash-attention/issues/453